### PR TITLE
Topics

### LINKED ISSUE #2007: [CMS] Topics
Topic properties:

- Banner image
- Thumbnail image
- Title
- Description
- MarketIds

### PR DESCRIPTION
**Pull request by yornaath**
- [x] Implement data fetching of topics from CMS
- [x] Implement front page topics section
- [x] Implement topics page

Fixes #2007

### CODE REVIEW

**Comment by mk26071980** [comment]
I like it overall. Spacing between the topics and the cards should be fixed though. And we can have the same hover effect as the cards, so topics don't blend into the background.
image

File: components/front-page/Topics.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            className={`
+            ztg-transition flex flex-1 cursor-pointer items-center gap-4 rounded-lg p-3 transition-all md:max-w-sm md:hover:scale-[1.015]
+              ${
+                (isString(selectedTopic) && selectedTopic === topic.slug) ||
+                (isObject(selectedTopic) && selectedTopic.slug === topic.slug)
+                  ? "bg-gray-200"
+                  : "bg-white"
+              }
+            `}
+            onClick={() => {
```
Comment (Robiquet) [inline components/front-page/Topics.tsx:36]: Could you use `next/link` instead to retain native browser functionality?

File: lib/cms/topics.ts [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  },
+) => {
+  const { markets } = await indexer.markets({
+    where: {
+      marketId_in: topic.marketIds,
+    },
+    limit: opts?.limit,
+  });
+
+  let marketCardsData = markets
```
Comment (Robiquet) [inline lib/cms/topics.ts:77]:
```suggestion
  const marketCardsData = markets
```

File: pages/topics/[topic].tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  params: { topic: string };
+}) {
+  const sdk = await create({
+    provider: endpointOptions.map((e) => e.value),
+    indexer: graphQlEndpoint,
+    storage: ZeitgeistIpfs(),
+  });
+
+  const cmsTopic = await getCmsFullTopic(params.topic);
+  let marketCardsData = await marketsForTopic(cmsTopic, sdk.indexer);
```
Comment (Robiquet) [inline pages/topics/[topic].tsx:40]:
```suggestion
  const marketCardsData = await marketsForTopic(cmsTopic, sdk.indexer);
```

**Review by Robiquet — COMMENTED** [review]
Looks good!

**Comment by Robiquet** [comment]
Looks like some of the card data isn't loading correctly 
image

File: components/front-page/Topics.tsx (hunk 1 above)
Comment (yornaath) [reply components/front-page/Topics.tsx]: Fixed

**Review by Robiquet — APPROVED** [review]

**Pull request merged**